### PR TITLE
Multiple warnings for the same code are not separated by new line characters

### PR DESCRIPTION
**Pull request by chlebowa**
Closes [this issue](https://github.com/insightsengineering/teal.code/issues/89).

Added line breaks at the end of every warning.
Added chevron at the beginning of every warning.
Slightly modified final output of `get_warnings`.

```
~~~Warnings~~~

> "ff" is not a graphical parameter
> "ff" is not a graphical parameter
> "ff" is not a graphical parameter
> "ff" is not a graphical parameter
when running code:
p <- hist(iris_data[, "Sepal.Length"], ff = "")

> this is a single warning
when running code:
warning("this is a single warning")

~~~Trace~~~

iris_data <- iris
p <- hist(iris_data[, "Sepal.Length"], ff = "")
warning("this is a single warning")
message("this is a single message")
```

### CODE REVIEW

File: R/qenv-get_warnings.R [hunk 1]
```diff
@@ -35,16 +35,18 @@ setMethod("get_warnings", signature = c("qenv"), function(object) {
...
-      warning_output <- paste(
-        warning_output, "\n>", warn, "\nWhen running code:\n", paste(object@code[warn_idx], collapse = "\n")
-      )
-    }
+  format_condition <- function(warn, expr) {
+    if (warn == "") return(NULL)
+    sprintf("%swhen running code:\n%s", warn, expr)
   }
-  paste0(warning_output, "\n\nTrace:\n", paste(get_code(object), collapse = "\n"))
+  lines <- mapply(format_condition,
```
Comment (gogonzo) [inline R/qenv-get_warnings.R:42]: no need to create a format_condition in the function environment as it's  used only in one place.

**Review by gogonzo — COMMENTED** [review]
Formats nicely. Please add tests

File: R/qenv-get_warnings.R (hunk 1 above)
Comment (chlebowa) [reply R/qenv-get_warnings.R:42]: That was for extra clarity, will change to anonymous function.

**Comment by chlebowa** [comment]
Tests raise this warning:
```
Warning (test-chunks.R:1547): overwrite_chunks
`init_chunks()` was deprecated in teal.code 0.2.1.
i Chunks are being deprecated qenv objects should be used instead
Backtrace:
 1. teal.code::init_chunks(session = session)
      at test-chunks.R:1547:2
```

Should tests for chunks be removed or skipped, or should these warnings be suppressed?

**Comment by nikolas-burkoff** [comment]
Ah there's a way to do handle the chunk tests nicely - i.e. to keep them but suppress the warning  https://lifecycle.r-lib.org/reference/verbosity.html

File: NEWS.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 ### Major breaking change
 * `chunks` have now been deprecated and will be removed from the package in a future release. The new `qenv` object should be used instead. See the new `qenv` vignette in the package for further details.
 
+### Enhancements
+* Improved output format of warnings in `get_warnings`.
```
Comment (gogonzo) [inline NEWS.md:7]: qenv hasn't been released yet, so improvement of `get_warnings` is not an item for a log change
```suggestion
```

**Review by gogonzo — APPROVED** [review]
Formats nicely and tests are fine

**Pull request merged**